### PR TITLE
support epsilla cloud vector database in langchain

### PR DESCRIPTION
**Pull request by eric-epsilla**
Description

- support epsilla cloud in langchain

### CODE REVIEW

File: libs/community/langchain_community/vectorstores/epsilla.py [hunk 1]
```diff
@@ -65,9 +65,9 @@ def __init__(
                 "Please install pyepsilla package with `pip install pyepsilla`."
             ) from e
 
-        if not isinstance(client, pyepsilla.vectordb.Client):
+        if not isinstance(client, pyepsilla.vectordb.Client) and not isinstance(client, pyepsilla.cloud.client.Vectordb):
```
Comment (leo-gan) [inline libs/community/langchain_community/vectorstores/epsilla.py:68]:
```suggestion
        if not isinstance(client, (pyepsilla.vectordb.Client, pyepsilla.cloud.client.Vectordb)):
```

Comment (eric-epsilla) [reply libs/community/langchain_community/vectorstores/epsilla.py]: Thanks for suggestion, committed

**Comment by eric-epsilla** [comment]
@leo-gan please help review, thanks

**Comment by leo-gan** [comment]
@eric-epsilla I would add a unit (or integration?) test to support the change.

**Comment by leo-gan** [comment]
LGTM

**Review by leo-gan — APPROVED** [review]
LGTM

**Review by ccurme — APPROVED** [review]

**Pull request merged**